scripts/ptx.js

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env node
```

Comment:
I think this is useful, but have the feeling it should belong to the Lambda Power Tuning repository since it doesn't have anything to do with the UI.

Reply:
All PTX-related files will be removed, as agreed :+1: